src/schemas/schemas.js

```diff
@@ -0,0 +1,14 @@
+const Joi = require('joi');
+
+const StudentSchema = Joi.object({
+  firstName: Joi.string().required(),
+  lastName: Joi.string().required(),
+  grade: Joi.number().required(),
+  language: Joi.string().required(),
+  parentsContacts: Joi.any().required(),
+  streamId: Joi.number().required(),
+  email: Joi.string().email(),
```

Comment:
Поля которые не обязательны нужно добавить что они могут быть пустыми
```suggestion
  email: Joi.string().email().allow(null, ''),
```

Reply:
@OmegonAlpharius Добавил и проверил

Reply:
@OmegonAlpharius  Добавил groupId в схему студента